docs/index.md

```diff
@@ -57,6 +57,7 @@ The following provider attributes are supported:
 - `realm` (Optional) - The realm used by the provider for authentication. Defaults to environment variable `KEYCLOAK_REALM`, or `master` if the environment variable is not specified.
 - `initial_login` (Optional) - Optionally avoid Keycloak login during provider setup, for when Keycloak itself is being provisioned by terraform. Defaults to true, which is the original method.
 - `client_timeout` (Optional) - Sets the timeout of the client when addressing Keycloak, in seconds. Defaults to environment variable `KEYCLOAK_CLIENT_TIMEOUT`, or 5 is the environment variable is not specified.
+- `verify_tls` (Optional) - Allows ignoring insecure certificates when set to false. Defaults to true. Disabling security check is dangerous and should be avoided.
```

Comment:
Would you mind renaming this to `tls_insecure_skip_verify` so it's clear that this should be avoided if possible?  Then we can update the default value to `false`

Reply:
Done in https://github.com/mrparkers/terraform-provider-keycloak/pull/237/commits/60df991d9aa63891ef8c86f6a11a18f575fa1ed5